packages/sdk/src/marketplaces/Opensea.ts

```diff
@@ -121,28 +122,28 @@ export class Opensea implements Marketplace<Order> {
     return createOrder(params);
   }
 
-  /* eslint-disable camelcase */
   async getOrders({
     makerAsset,
     maker,
     takerAsset,
     taker,
   }: GetOrdersParams = {}): Promise<Order[]> {
-    const query: OrderQuery = {
+    const query: Omit<OrdersQueryOptions, "limit"> = {
+      // https://github.com/ProjectOpenSea/opensea-js/blob/master/src/api.ts#L106 limit is currently omitted here
+      protocol: "seaport",
       maker,
-      sale_kind: SaleKind.FixedPrice,
       taker,
+      side: ORDER_SIDE_SELL,
```

Comment:
Side is now required param, so I'm providing default value here, since without it TS gets tricky below with conditional assignments and object spreads.